crates/stc_ts_type_checker/tests/conformance/jsx/tsxAttributeResolution5.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
     required_error: 3,
     matched_error: 0,
-    extra_error: 0,
+    extra_error: 1,
```

Comment:
The error about err_shift_n

Reply:
I don't think this is a problem of `err_shift_n`

Reply:
This is actually a regression.
I checked it

Reply:
But well, let's ignore this for now